frontend/src/components/Room.tsx

```diff
@@ -94,20 +103,67 @@ function Room() {
     } ));
 
 
+    const [roomAnnotation, setRoomAnnotation ] = useState(room?.roomAnnotation===null ? "" : String(room?.roomAnnotation))
```

Comment:
the best way is to put useStates on top of the component, so that we know what might change